app/src/main/java/com/example/campusguide/Bootstrapper.kt

```diff
@@ -7,6 +7,7 @@ import com.example.campusguide.location.FusedLocationProvider
 import com.example.campusguide.location.SwitchCampus
 import com.example.campusguide.map.GoogleMapAdapter
 import com.example.campusguide.map.GoogleMapInitializer
+import com.example.campusguide.search.CustomSearch
 import com.example.campusguide.search.Search
```

Comment:
import not being used. But i guess You're leaving this in case we revert back to the Search rather than CustomSearch?

Reply:
May aswell remove the import until we need it again

Reply:
Removed (we can re-add it again if we need it)